ocdaction/ocdaction/templates/challenge/challenge_add.html

```diff
@@ -1,17 +1,19 @@
 {% extends "layout.html" %}
 
+{% block title %}Act: Add new challenge{% endblock %} 
+
```

Comment:
We don't use 'Act' in challenges context anymore

Reply:
Should I remove the h1 then that says ACT?